src/test/shell/testenv.sh

```diff
@@ -296,12 +296,6 @@ build --incompatible_skip_genfiles_symlink=false
 
 build --incompatible_use_toolchain_resolution_for_java_rules
 
-# Enable Bzlmod in all shell integration tests
-common --enable_bzlmod
-
-# Disable WORKSPACE in all shell integration tests
-common --noenable_workspace
-
```

Comment:
Removed as these flags are no longer necessary (they agree with defaults), produce deprecation warnings in tests, and in (at least) one case the deprecation warnings broke a test (more `stdio` output than expected).